mozilla-release/mobile/android/chrome/content/browser.js

```diff
@@ -6892,6 +6893,11 @@ var Cliqz = {
           name: 'updateBlocking',
           message: blockingPolicy
         });
+        break;
+      case "Search:ChangeTheme":
+        const theme = data.isLightTheme ? "white" : "blue";
```

Comment:
This code is already in sync with the extension I guess, right?

Reply:
Hi, @giota-cliqz. Should be, as it seems to apply the changes.